doc/content/docs/imjoy.md

```diff
@@ -5,16 +5,24 @@ title: ImJoy Plugin
 
 An *itk-vtk-viewer* plugin is available for [ImJoy](https://imjoy.io), a plugin powered hybrid computing platform for deploying deep learning applications such as advanced image analysis tools.
 
+![ImJoy itk-vtk-viewer plugin](./imjoy.png)
+
 ## Installation
 
-Install the plugin into the workspace with the following [ImJoy Web App](http://imjoy.io/#/app?plugin=https://kitware.github.io/itk-vtk-viewer/app/) or [ImJoy Lite App](http://imjoy.io/#/lite?plugin=https://kitware.github.io/itk-vtk-viewer/app/) links with the plugin URI:
+Install the plugin into the workspace with the following [ImJoy Web App](http://imjoy.io/#/app?plugin=https://kitware.github.io/itk-vtk-viewer/app/) or [ImJoy Lite App](http://imjoy.io/lite?plugin=https://kitware.github.io/itk-vtk-viewer/app/) links with the plugin URI:
 
 ```
 https://kitware.github.io/itk-vtk-viewer/app/
 ```
 
 Note that the link can also be used directly.
 
+To install a specific version, e.g. version `10.8.0`, use the URI:
+
+```
+https://unpkg.com/itk-vtk-viewer@10.8.0/dist/index.html
```

Comment:
Didn't know unpkg serves static site, this is a really cool feature for reproducibility. I was using netlify PR previews, but it is limited for build time for free account.

Reply:
The unpkg CDN is a nice, automatic bonus that comes when publishing to NPM -- yes, specific versions should help with reproducibility :-)